carbonmark-api/test/routes/vintages/get.test.ts

```diff
@@ -1,16 +1,22 @@
 import { FastifyInstance } from "fastify";
 import nock from "nock";
 import { aProject } from "../../../src/.generated/mocks/marketplace.mocks";
-import { GRAPH_URLS } from "../../../src/app.constants";
+import { GRAPH_URLS, ICR_API } from "../../../src/app.constants";
+import { COUNTRY_CODES, VINTAGES } from "../../../test/fixtures/icr";
 import { build } from "../../helper";
 import { DEV_URL, ERROR } from "../../test.constants";
 
 describe("GET /vintages", () => {
   let fastify: FastifyInstance;
+  let ICR_API_URL: string;
 
   // Setup the server
   beforeEach(async () => {
+    const icrApiValues = ICR_API("polygon");
+    ICR_API_URL = icrApiValues.ICR_API_URL;
+
     fastify = await build();
+    nock.cleanAll();
```

Comment:
this shouldnt be necessary because it's already happening in the `build` fn

Reply:
removed